tests/ui/patterns.rs

```diff
@@ -1,8 +1,10 @@
 #![allow(unused)]
```

Comment:
```suggestion
// run-rustfix
#![allow(unused)]
```

Reply:
Don't forget to run `cargo uitest` and `update-all-references.sh` after that

Reply:
Okay, added the flag and `.fixed` file.